### PR TITLE
Fix for Python 4

### PR DESCRIPTION
**Pull request by hugovk**
We don't yet know if 3.10 or 4.0 will follow Python 3.9, but whichever it is, it will probably happen in [2020 when Python 3.9 reaches beta](https://www.python.org/dev/peps/pep-0596/#schedule) and work begins on Python 3.9+1.

There's some code which uses `six.PY3`:

```python
if six.PY3:
    long = int
```

Where:

```python
PY3 = sys.version_info[0] == 3
```

When run on Python 4, this skip the line! Instead, use `not six.PY2`.

Found using https://github.com/asottile/flake8-2020:
```console
$ pip install -U flake8-2020
$ flake8 --select YTT
```

### CODE REVIEW

File: pytest_bdd/cucumber_json.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 from .feature import force_unicode
 
-if six.PY3:
+if not six.PY2:
```
Comment (youtux) [inline pytest_bdd/cucumber_json.py]: Let's instead remove usages of `long` completely and replace them with `int`.

**Comment by youtux** [comment]
The CI build is failing because `attrs` 19.2.0 was recently released, and it has some changes that are incompatible with pytest between 4.0 and 4.2.
We should probably drop compatibility with pytest < 4.3 then.

**Comment by hugovk** [comment]
Thanks, updated!

**Comment by youtux** [comment]
Can you also add an entry to the changelog? Thanks

**Comment by hugovk** [comment]
You're welcome!

**Pull request merged**